waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/impl/StaticDatabaseMappingService.java

```diff
@@ -155,6 +162,18 @@ private void validateFederatedMetastoreDatabases(List<String> mappableDatabases,
     }
   }
 
+  private List<String> getDatabasesFromPattern(Iface client, List<String> mappedDatabases) {
+    List<String> matchedDatabases = new ArrayList<String>();
+    for (String mappedDatabase : mappedDatabases) {
+      try {
+        matchedDatabases.addAll(client.get_databases(mappedDatabase));
+      } catch (TException e) {
+        LOG.error("Could not match databases for '{}'", mappedDatabase, e);
```

Comment:
This, for now, logs an error if the pattern/database name doesn't match with anything in the metastore. Should it actually stop the execution if the database regex/name is incorrect and no database is found?

Reply:
No if the pattern doesn't match we just don't include anything. For instance consider someone dropping a database that was configured before. We don't want WD to fail if that happens.